task-docs/TEST.md

```diff
@@ -0,0 +1,27 @@
+# `test`
+
+```
+{
+	interactive: false,
+	flags: [
+		--browserstack,
```

Comment:
it would be nice if it could prompt for username & pass if the `BROWSER_STACK_` environment variables aren't set

Reply:
(rather than exiting angrily like `obt`)